Extensions/dnSpy.Debugger/dnSpy.Debugger/ToolWindows/CodeBreakpoints/CodeBreakpointsVM.cs

```diff
@@ -543,5 +539,30 @@ sealed class CodeBreakpointVMComparer : IComparer<CodeBreakpointVM> {
 			codeBreakpointContext.UIDispatcher.VerifyAccess();
 			FilterText = string.Empty;
 		}
+
+		// UI thread
+		public void SortList_UI() {
+			codeBreakpointContext.UIDispatcher.VerifyAccess();
+			var newList = new List<CodeBreakpointVM>(GetFilteredItems_UI(filterText));
+			newList.Sort(SortListComparer);
+			AllItems.Reset(newList);
+		}
+		private CodeBreakpointVMComparer sortListComparer;
+		public CodeBreakpointVMComparer SortListComparer {
+			get {
+				if (sortListComparer == null) sortListComparer = CodeBreakpointVMComparer.Instance;
+				return sortListComparer;
+			} set {
```

Comment:
Formatting problems. Select your code in VS and `Ctrl+K, Ctrl+F`, should fix some of the formatting issues in all your files.